### PR TITLE
fix(mechanics): Correctly parse `add description/spaceport` text nodes

### PR DESCRIPTION
**Pull request by tibetiroka**
**Bug fix**

This PR addresses the bug/feature described in https://github.com/endless-sky/endless-sky/issues/10309#issuecomment-2220679738

## Summary
The text is always the last token, so just parse that.

## Testing Done
Warp's provided save now works, both with and without `add`.

## Save File
This save file can be used to test these changes:
[warp.core.add.descriptioin.tes.txt](https://github.com/user-attachments/files/16189347/warp.core.add.descriptioin.tes.txt)

### CODE REVIEW

**Review by warp-core — COMMENTED** [review]
Should the case for a node with a child "to display" node not also support "add"?

**Comment by tibetiroka** [comment]
I don't see a reason why it shouldn't, though it's not necessary either.

**Comment by warp-core** [comment]
> I don't see a reason why it shouldn't, though it's not necessary either.

It just seems a bit strange that:
```
planet "Wherever"
    description "The initial description."

planet "Wherever"
    add description "More description."
```
Will work (after this PR) but:
```
planet "Wherever"
    description "The initial description."

planet "Wherever"
    add description "More description."
        to display
            has "but only sometimes"
```
Will not work (unchanged by this PR) (it'll be adding the word "description", instead of the actual description text.)

That seems like something that should be added here as well, unless for some reason we don't want that to work?

File: source/Paragraphs.cpp [hunk 1]
```diff
@@ -31,7 +31,7 @@ void Paragraphs::Load(const DataNode &node)
 			text.emplace_back(child, node.Token(1) + "\n");
```
Comment (warp-core) [inline source/Paragraphs.cpp:31]:
```suggestion
			text.emplace_back(child, node.Token(node.Size() - 1) + "\n");
```

**Comment by tibetiroka** [comment]
Oh, I didn't realize it was a missing fix.

**Pull request merged**